optimade/server/config.py

```diff
@@ -137,23 +137,15 @@ def set_implementation_version(cls, v):
         res.update(v)
         return res
 
-    @validator("log_level")
-    def force_debug_log_level(cls, v, values):
-        """If `debug` is `True`, then force `log_level` to be `DEBUG` as well"""
-        from optimade.server.logger import CONSOLE_HANDLER
-
-        if values.get("debug", False):
-            v = LogLevel.DEBUG
-        CONSOLE_HANDLER.setLevel(v.value.upper())
-        return v
-
     @root_validator(pre=True)
     def load_default_settings(cls, values):  # pylint: disable=no-self-argument
         """
         Loads settings from a root file if available and uses that as defaults in
         place of built in defaults
         """
-        from optimade.server.logger import LOGGER
+        import logging
```

Comment:
Since it's not registering / using the set up logger, maybe just remove logging completely here?

Reply:
Instead, perhaps, once during startup of the server it logs what configuration file it's using, if any.

Reply:
Think I'd prefer that, yeah